ui/app/templates/components/job-page/service.hbs

```diff
@@ -6,6 +6,7 @@
     <jobPage.ui.DasRecommendations />
     <jobPage.ui.StatusPanel @statusMode={{@statusMode}} @setStatusMode={{@setStatusMode}} />
     <jobPage.ui.PlacementFailures />
+    {{!-- latestDeployment only included here for visual comparison during build-out --}}
```

Comment:
^---- later along this epic, the latestDeployment component can be safely deleted from the codebase.